### PR TITLE
#61 catch and report exception: parsing + http + eval javascript

### PR DESCRIPTION
**Pull request by npeters**
I try to add a call to TestReportStore on each catch.
I add a test with a javascript error.

### CODE REVIEW

**Comment by quangson91** [comment]
@npeters  sorry for the late reply.
Please  check the unit test. 
Also, it would be good if the code is formatted.
I will add the ktlint checker in the future.

File: src/main/java/uos/dev/restcli/HttpRequestFilesExecutor.kt [hunk 1]
```diff
@@ -73,6 +73,12 @@ class HttpRequestFilesExecutor constructor(
             parser.parse(httpFilePath)
         } catch (e: Exception) {
             logger.error(e) { "Can't parse $httpFilePath" }
+            val trace = TestGroupReport.Trace(
+                httpTestFilePath = httpFilePath,
+                scriptHandlerStartLine = -1
+            )
+            TestReportStore.addTestGroupReport("-", trace)
+            TestReportStore.addTestReport("Parsing",false, e.message,e.message)
```
Comment (quangson91) [inline src/main/java/uos/dev/restcli/HttpRequestFilesExecutor.kt:81]: Please format code before commit.
For example,
`"Parsing",false` => `"Parsing", false`

**Comment by npeters** [comment]
I setup my IDEA with  `ktlint applyToIDEAProject`,
the code should be format well now ;)

**Comment by quangson91** [comment]
@npeters  Great!

**Pull request merged**